packages/manager/src/features/CloudPulse/shared/CloudPulseRegionSelect.tsx

```diff
@@ -61,17 +62,17 @@ export const CloudPulseRegionSelect = React.memo(
         (item: CloudPulseResourceTypeMapFlag) =>
           item.serviceType === serviceType
       );
-      const supportedRegionsIdList =
-        resourceTypeFlag?.supportedRegionIds
-          ?.split(',')
-          .map((regionId: string) => regionId.trim())
-          .filter((regionId: string) => regionId.length > 0) || [];
 
-      if (!supportedRegionsIdList.length) {
+      if (isNil(resourceTypeFlag?.supportedRegionIds)) {
```

Comment:
hope it does not validate the length of the string

Reply:
only checks for null/undefined, works as expected